.redocly.lint-ignore.yaml

```diff
@@ -0,0 +1,160 @@
+# This file instructs Redocly's linter to ignore the rules contained for specific parts of your API.
```

Comment:
Can we add a link to this as a comment somewhere in this document please?
https://github.com/Redocly/redocly-cli/issues/925#issuecomment-2355005587